storage/mounts_test.go

```diff
@@ -21,3 +25,92 @@ func TestValidateMountDirectory(t *testing.T) {
 		assert.Lenf(t, errs, tc.errorCount, "directory '%s' does not generate the expected error count", tc.directory)
 	}
 }
+
+var expectedCommandResp = "done"
+
+func testMountFuncHelper(t *testing.T, mountName, expectedCommand string, mount Mount,
+	mountFunc func(mp MountPoint, mount Mount) (string, error)) {
+	c := common.DatabricksClient{
+		Host:  ".",
+		Token: ".",
+	}
+	err := c.Configure()
+	assert.NoError(t, err)
+
+	var called bool
+
+	c.WithCommandMock(func(commandStr string) (s string, e error) {
+		called = true
```

Comment:
it may be called few different times with different commands, so keep this consideration in tests

Reply:
refactored tests to only test mp.Mount, Source and Delete which only invoke this once.